recipes/octo-keygen-cpp/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
```

Comment:
```suggestion
from conans import ConanFile, CMake
from conan.tools.build import cross_building
```
Linter will not accept cross_building from conans

Reply:
fixed :)